### PR TITLE
Add unassign feature

### PR DESCRIPTION
**Pull request by marianeum**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
 name: 'Find reviewers'
 description: "Find random reviewers"
+inputs:
+  token:
+    description: 'Github token'
+    required: true
+  type:
+    description: 'Event that triggered this action'
+    required: true
```
Comment (Crunch09) [inline action.yml:9]: That probably caused the warning we were seeing. Thanks for adding!

File: action.yml [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 outputs:
   time:
     description: "The time this command ran"
```
Comment (Crunch09) [inline action.yml:12]: Understand that this wasn't added by your PR but feel free to remove this `output`

File: package.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "@octokit/rest": "^18.0.0",
     "@slack/webhook": "^5.0.4",
     "@types/node-fetch": "^2.5.7",
+    "tsc": "^1.20150623.0",
```
Comment (Crunch09) [inline package.json:30]: I think we only want this in the dev-dependencies. You can install it in there with `npm install --save-dev tsc`

File: .github/find_reviewers.yml [hunk 4]
```diff
@@ -8,22 +8,12 @@ labels:
...
         number_of_picks: 1
         possible_reviewers:
-          - romain
-          - maria
-          - amal
-          - ana
+          - marianeum
+          - Crunch09
+          - jabeattie
     label: "documentation"
```
Comment (Crunch09) [inline .github/find_reviewers.yml:17]: Think we should update this from `documentation` as well and use something like `ready-for-review`

**Review by Crunch09 — APPROVED** [review]
Looks good, just a few comments. Thank you!

File: package.json (hunk 3 above)
Comment (marianeum) [reply package.json:30]: ah sorry! i'll remove these :)

**Pull request merged**